torchvision/ops/_custom_ops.py

```diff
@@ -0,0 +1,44 @@
+import os
+import torch
+import glob
+
+# load the custom_op_library and register the custom ops
+custom_op_lib_path = os.path.join(os.path.dirname(__file__), '..', 'custom_ops.*.so')
```

Comment:
This only works for Linux, and we should find a way to make it work for OSX / Windows as well.

Not sure if we have a built-in tool for that in PyTorch.

Reply:
I don't think we have this properly yet, but maybe we should.